package.json

```diff
@@ -26,6 +26,7 @@
     "react-native": "0.66.0",
     "react-native-autocomplete-input": "^5.0.2",
     "react-native-datawedge-intents": "^0.1.8",
+    "react-native-dropdown-picker": "^5.2.3",
```

Comment:
This is definitely not the lib that was requested. In the title you have `react-native-paper-dropdown`